openedx/features/edly/context_processor.py

```diff
@@ -38,6 +40,10 @@ def dynamic_theming_context(request):  # pylint: disable=unused-argument
         theming_context.update(
             {'edly_branding_config': configuration_helpers.get_dict('BRANDING', DEFAULT_BRANDING_DICT)}
         )
+        theming_context.update(
+            {'services_notifications_url': configuration_helpers.get_value('SERVICES_NOTIFICATIONS_URL',
```

Comment:
I have only one concern, that this change is not related to theming.
Maybe we can have a new edly context processor `edly_app_context` where we can provide this value.

Reply:
These changes are required in edly themes and this is a custom context processer we added for our themes. Kindly suggest, should we write a separate context processor for this URL or not.

Reply:
I think since it's a very small change, it's fine if it's done in this context processor. if a bigger change comes, we can make a separate context processor. Otherwise whatever zubair bhai thinks.

Reply:
As per discussion with @zubair-arbi we will add separate this context processor in a separate ticket.

Reply:
Feedback Implemented